Secrets/KeysExample.swift

```diff
@@ -4,3 +4,5 @@ struct Keys {
         static let clientKey = ""
     }
 }
+
+let x = 1
```

Comment:
Name Format Violation (Medium Severity): Variable name should be between 3 and 40 characters in length: 'x'